resources/views/auth/authorize.blade.php

```diff
@@ -4,94 +4,114 @@
 
-                        <ul>
-                            @foreach ($webhook['events'] as $event)
-                            <li>{{ __("settings.webhook_event." . $event) }}</li>
-                            @endforeach
-                        </ul>
-                    </div>
-                    @endif
+                                <ul>
+                                    @foreach ($webhook['events'] as $event)
+                                        <li>{{ __("settings.webhook_event." . $event) }}</li>
+                                    @endforeach
+                                </ul>
+                            </div>
+                        @endif
 
-                    <div class="d-flex justify-content-evenly">
-                        <!-- Authorize Button -->
-                        <form method="post" action="{{ route('oauth.authorizations.approve') }}">
-                            @csrf
+                        @if($client->user_id !== 0)
+                            <div class="alert alert-danger">
+                                <i class="fas fa-warning flex-shrink-0 me-2 bi"></i>
+                                {{ __("menu.oauth_authorize.third_party") }}
+                                <a href="https://help.traewelling.de/safety-and-security/apps/#drittanbieter-apps"
+                                   target="_blank">
+                                    {{ __("menu.oauth_authorize.third_party.more") }}
+                                </a>
+                            </div>
+                        @endif
 
-                            <input type="hidden" name="state" value="{{ $request->state }}">
-                            <input type="hidden" name="client_id" value="{{ $client->id }}">
-                            <input type="hidden" name="auth_token" value="{{ $authToken }}">
-                            <button type="submit" class="btn btn-success btn-approve">{{__("menu.oauth_authorize.authorize")}}</button>
-                        </form>
+                        <div class="d-flex justify-content-evenly">
+                            <!-- Authorize Button -->
+                            <form method="post" action="{{ route('oauth.authorizations.approve') }}">
+                                @csrf
 
-                        <!-- Cancel Button -->
-                        <form method="post" action="{{ route('passport.authorizations.deny') }}">
-                            @csrf
-                            @method('DELETE')
+                                <input type="hidden" name="state" value="{{ $request->state }}">
+                                <input type="hidden" name="client_id" value="{{ $client->id }}">
+                                <input type="hidden" name="auth_token" value="{{ $authToken }}">
+                                <button type="submit"
+                                        class="btn btn-success btn-approve">{{__("menu.oauth_authorize.authorize")}}</button>
```

Comment:
formatting (long line)
```suggestion
                                <button type="submit" class="btn btn-success btn-approve">
                                        {{__("menu.oauth_authorize.authorize")}}
                                </button>
```